models/models.go

```diff
@@ -851,22 +851,26 @@ func (m *RouteManager) GetDNSInstructions(route *Route) ([]string, error) {
 		"instance-id": route.InstanceId,
 	})
 
+	lsession.Info("load-user")
 	user, err := route.loadUser(m.db)
 	if err != nil {
-		lsession.Error("load-user", err)
+		lsession.Error("load-user-err", err)
 		return instructions, err
 	}
 
 	if err := json.Unmarshal(route.ChallengeJSON, &challenges); err != nil {
 		lsession.Error("json-unmarshal-challenge", err)
```

Comment:
In [`ensureChallenges`](https://github.com/alphagov/paas-cdn-broker/pull/16/commits/643d6e56d10cefdb8d70298e813cdbeaa60689be#diff-0b230c6239e29d9787ee9d482d175728R825) we logged a message before unmarshalling. I think we should be consistent about that either way (adding here, or removing it there)

Reply:
Removed the message from `ensureChallenges` as it didn't have much value on top of the error-case logging.